exchanges/okx/okx_websocket.go

```diff
@@ -554,8 +515,14 @@ func (ok *Okx) WsHandleData(respRaw []byte) error {
 		}
 		return fmt.Errorf("%w unmarshalling %v", err, respRaw)
 	}
-	if (resp.Event != "" && (resp.Event == "login" || resp.Event == "error")) || resp.Operation != "" {
-		ok.WsResponseMultiplexer.Message <- &resp
+	if resp.Event != "" &&
+		(resp.Event == "login" ||
+			(resp.Event == "error" &&
+				slices.Contains([]string{"60022", "60023", "60024", "60026", "63999", "60032", "60011", "60009", "60005", "60021", "60031"}, resp.Code))) {
```

Comment:
🚨 🚑 🚨 
What kind of insane slice ordering is this? Its not like you were going down the list in okx docs and adding them 😆 
Would 6007 fit?